settings.py

```diff
@@ -1191,6 +1191,12 @@ def _pkce_required(client_id):
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = SECURE_COOKIES = not DEBUG
 SESSION_COOKIE_HTTPONLY = CSRF_COOKIE_HTTPONLY = True
 
+# This is commented because it is not required now. We don't need to instrument all the services rn on staging.
+# The below lines can be uncommented when we need to turn on app level tracing on any env.
+# if SERVER_ENVIRONMENT == 'staging':
+#     from ddtrace import patch_all
+#     patch_all()
```

Comment:
Even though this is now commented out, I don't think it should go in _settings.py_. I think it could go in _manage.py_ or some other module (or preferably a function) that is called early during startup.

Reply:
> Even though this is now commented out, I don't think it should go in settings.py

This was a recommendation by Datadog, they want it to be in `settings.py` ideally or else it should be happening as early as possible during startup so that the required modules can be patched.

Reply:
I don't think this follows best practices since it's not ideal to have side effects like this as a result of importing a module. Not going to block this PR on that since it's Datadog's recommendation.